packages/lesswrong/components/events/modules/VirtualProgramCard.tsx

```diff
@@ -104,30 +104,17 @@ const VirtualProgramCard = ({program, classes}: {
 }) => {
   const { captureEvent } = useTracking()
   
-  // find the next deadline for applying to the Intro VP, which is the last Sunday of every month
-  let deadline = moment()
-  // Iterate through 5 Sundays to find the one we want
-  for (const sunday of _.range(5).map(x => moment().day(0).add(x, 'week'))) {
-    const nextSunday = moment(sunday).add(1, 'week')
-    // needs to be in the future, and needs to be the last Sunday of the month
-    if (sunday.isSameOrAfter(moment(), 'day') && nextSunday.month() !== sunday.month()) {
-      deadline = sunday
-      break
-    }
-  }
-  // this Dec, the dates are different
-  // TODO: remove once 2023 arrives
-  const dec11 = moment("12-11-2022", "MM-DD-YYYY") // deadline for Jan-Feb cohort (start date is Jan 9)
-  const jan1 = moment("01-01-2023", "MM-DD-YYYY")
-  const jan29 = moment("01-29-2023", "MM-DD-YYYY") // deadline for Feb-Mar cohort? (should be normal start date logic)
-  if (moment().isSameOrBefore(dec11)) {
-    deadline = dec11
-  } else if (moment().isSameOrBefore(jan1)) {
-    deadline = jan29
+  // Find the next deadline for applying to the Intro VP, which is usually the 4th Sunday of every month
+  // (though it will sometimes move to the 5th Sunday - this is not accounted for in the code).
+  // This defaults to the Sunday in the week of the 28th day of this month.
+  let deadline = moment().date(28).day(0)
```

Comment:
That is a stupid name for the dayOfWeek function

Reply:
Agree